nixos/modules/services/networking/minidlna.nix

```diff
@@ -36,6 +36,36 @@ in
         '';
     };
 
+    services.minidlna.friendlyName = mkOption {
+      type = types.str;
+      default = "${config.networking.hostName} MiniDLNA";
```

Comment:
I think you'll need a `defaultText` field as well containing something like
```
defaultText = "$HOSTNAME MiniDLNA"
```
otherwise the manual would have to be rebuilt for each host.

Reply:
ok